ethpm/contract.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, address: bytes, **kwargs: Any) -> None:
                 "Contract cannot be instantiated until its bytecode is linked."
             )
         validate_address(address)
-        super(LinkableContract, self).__init__(address=address, **kwargs)
```

Comment:
Doesn't seem like it would be ok to drop the `**kwargs` here.  That would silently ignore any additional parameters beyond `address` (like `abi`, `bytecode`)?